galley/pkg/config/analysis/README.md

```diff
@@ -98,7 +98,7 @@ If your analyzer requires any new message types (meaning a unique template and e
 
     ```sh
     cd galley/pkg/config/analysis/msg
-    go generate
+    REPO_ROOT=$(git rev-parse --show-toplevel) go generate
```

Comment:
This is actually not the way to generate files anymore.

Instead, you go to the top of the repo and do:

BUILD_WITH_CONTAINER=1 make gen

Soon you won't need the BUILD_WITH_CONTAINER part, but for now you do.

Reply:
I initially had that but threw it out because it's so slow and I wasn't sure it was the recommended way.

Updated so that it mentions the right command.